parse_file.js

```diff
@@ -0,0 +1,12 @@
+const fs = require("fs");
+
+let ResultArray;
+fs.readFile("./Assignment.txt", "utf8", function (err, contents) {
+  if(err) throw new Error(err)
+  ResultArray = contents.split(/Q[0-9]+\)/);
+  var file = fs.createWriteStream("New_Assignment.txt");
+  ResultArray.forEach((line, index) => {
+    if(index) file.write("Q" + index + ") " + line);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/AbhishekSharma74542/Ozivatask/pullRequest?prid=8835547)
```suggestion
    if(index) {file.write("Q" + index + ") " + line);}
```